test/unit/reductions/exponentiated_gradient/test_expgrad_arguments.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pandas as pd
+import pytest
+
+
+from fairlearn.reductions import ExponentiatedGradient
+from fairlearn.reductions import DemographicParity
+from fairlearn.reductions import ErrorRate
+from .simple_learners import LeastSquaresBinaryClassifierLearner
+from .test_utilities import sensitive_features, X1, X2, X3, labels
+
+from test.unit.input_convertors import conversions_for_1d, ensure_ndarray, ensure_dataframe
+
+# ===============================================================
+
+# Ways of transforming the data
+candidate_X_transforms = [ensure_ndarray, ensure_dataframe]
+candidate_Y_transforms = conversions_for_1d
+candidate_A_transforms = conversions_for_1d
+
+# ================================================================
+
+
+class TestExpGradArguments:
```

Comment:
nit: we should probably try not to use "expgrad" since the method is fully spelled out elsewhere, just for consistency and to avoid confusing users.